rust/crd/src/authentication.rs

```diff
@@ -0,0 +1,115 @@
+use crate::ObjectRef;
+
+use serde::{Deserialize, Serialize};
+use snafu::{ResultExt, Snafu};
+use stackable_operator::commons::authentication::AuthenticationClassProvider;
+use stackable_operator::{
+    client::Client,
+    commons::authentication::AuthenticationClass,
+    schemars::{self, JsonSchema},
+};
+
+const SUPPORTED_AUTHENTICATION_CLASS_PROVIDER: [&str; 1] = ["TLS"];
```

Comment:
```suggestion
const SUPPORTED_AUTHENTICATION_CLASS_PROVIDERS: [&str; 1] = ["TLS"];
```
if we name it this way, when we add a new provider, we don't need to rename the variable all over the place

Reply:
done